client/web/src/enterprise/batches/batch-spec/edit/workspaces-preview/WorkspacesPreview.tsx

```diff
@@ -358,3 +369,36 @@ const CTASizeWarning: React.FunctionComponent<React.PropsWithChildren<{ totalCou
         you can.
     </Alert>
 )
+
+const NoExecutorsWarning: React.FunctionComponent<React.PropsWithChildren<{}>> = () => {
+    const [isRunServerSideModalOpen, setIsRunServerSideModalOpen] = useState(false)
+
+    return (
+        <div className={styles.container}>
+            <WorkspacesListHeader>Workspaces preview</WorkspacesListHeader>
+            <div className="d-flex flex-column align-items-center w-100 mb-3 px-5">
+                <Icon
+                    aria-label="The workspaces previewed below may not be up-to-date."
+                    className={styles.noExecutorsWarningIcon}
+                    inline={false}
+                    height="2.4rem"
+                    width="2.4rem"
+                    svgPath={mdiAlert}
+                />
+                <H4 className={styles.instruction}>
+                    Workspaces preview is only available when running batch changes server-side is enabled.{' '}
```

Comment:
I know this is echoing the title above, but this feels less grammatical in a sentence to me. How about:

```suggestion
                    Workspaces can only be previewed when running batch changes server-side is enabled.{' '}
```